db/migrate/20241101231144_change_final_weight_volume_to_scale_3_in_spree_line_items.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeFinalWeightVolumeToScale3InSpreeLineItems < ActiveRecord::Migration[7.0]
+  def up
+    change_column :spree_line_items, :final_weight_volume, :decimal, precision: 10, scale: 3
+  end
+
+  def down
+    change_column :spree_line_items, :final_weight_volume, :decimal, precision: 10, scale: 2
```

Comment:
Just noting that this action would be destructive.. but I guess if somebody was running the `down` action they would be expecting that.